### DISCUSSION TITLE
Proofreading of the specification

### DISCUSSION THREAD

**Discussion by hchiba1** [pg-format/pg-formatter#63] [General]

**Comment by hchiba1** [comment]
**1.2 Terminology**
"SHOULD NOT" ("RECOMMENDED") is not used in the spec.

**Reply by nichtich** [reply]
Not a problem, leaving it out might confuse as well, so keeping it.

**Comment by hchiba1** [comment]
**1.1 Motivation**
`an abstract data structure`
=> or just `a data structure`?
(Is `abstract` used to emphasize that the data structre does not depend on a specific implementation?)

**Reply by nichtich** [reply]
A better term may be "data model" but at this place it's just a description of current practice, so I'd change this to a verbal form:

> **Property Graphs** (also known as **Labeled Property Graphs**) are used to structure data in graph databases and related applications.

**Comment by hchiba1** [comment]
**1.1 Motivation**
`Hirokazu Chiba, Ryota Yamanaka, and Shota Matsumoto (2019, 2022)`
The G2GML paper was published in ISWC 2020, and then the same manuscript was shared on arXiv.
Thus, the latter one `2022` would be `2020`.

Ref. https://scholar.google.co.jp/scholar?cluster=15907590350345919988&hl=ja&as_sdt=0,5

**Reply by nichtich** [reply]
done

**Comment by hchiba1** [comment]
**3.9 Grammar**
I prefer `PlainKey` instead of `UnquotedKeyCol` (which is harder to get the meaning),
although it does not affect the compatibility issues.
